src/services/room-service.js

```diff
@@ -77,4 +77,8 @@ export default class RoomService {
   getRoomById(roomId) {
     return this.roomStore.findOne({ _id: roomId });
   }
+
+  isRoomMemberOrOwner(roomId, userId) {
+    return this.roomStore.findOne({ $or: [{ '_id': roomId, 'members.userId': userId }, { _id: roomId, owner: userId }] });
```

Comment:
Given the function name I'd expect a real boolean

Reply:
I had big trouble with naming this and that was my problem with this name as well, I'm just afraid we will end up having this same query in another context. I'll change it to returning a boolean.

Reply:
Fixed

Reply:
Thanks